inventory/sample/group_vars/k8s_cluster/addons.yml

```diff
@@ -139,6 +139,8 @@ metallb_enabled: false
 metallb_speaker_enabled: true
 # metallb_ip_range:
 #   - "10.5.0.50-10.5.0.99"
+# metallb_pool_name: "loadbalanced"
+# matallb_auto_assign: false
```

Comment:
Could you specify the default value `true` here?
The sample inventory contains the default values as possible.

Reply:
@oomichi thanks for the adjustment! Fixed.